[FEATURE] Ajout carousel nouvelle version

JTEPIX opened the pull request:
## 🔆 Problème

<!-- Décrivez ici le besoin ou l'intention couvert par cette Pull Request. -->

## ⛱️ Proposition

<!-- Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés. -->

## 🌊 Remarques

<!-- Des infos supplémentaires, trucs et astuces ? -->

## 🏄 Pour tester
https://app-pr13282.review.pix.fr/modules/tmp-prompt-intermediaire/details

JTEPIX committed:
Ajout carousel nouvelle version

pix-bot-github commented:
Choisir les applications à déployer :

- [x] [API](https://api-pr13282.review.pix.fr/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr13282/environment) <!-- pix-api-review -->
- [x] [App (.fr)](https://app-pr13282.review.pix.fr) – [App (.org)](https://app-pr13282.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-app-review-pr13282/environment) <!-- pix-app-review -->
- [ ] [Orga (.fr)](https://orga-pr13282.review.pix.fr) – [Orga (.org)](https://orga-pr13282.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-orga-review-pr13282/environment) <!-- pix-orga-review -->
- [ ] [Certif (.fr)](https://certif-pr13282.review.pix.fr) – [Certif (.org)](https://certif-pr13282.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-certif-review-pr13282/environment) <!-- pix-certif-review -->
- [ ] [Junior](https://junior-pr13282.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-junior-review-pr13282/environment) <!-- pix-junior-review -->
- [ ] [Admin](https://admin-pr13282.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-admin-review-pr13282/environment) <!-- pix-admin-review -->
- [ ] [API MaDDo](https://pix-api-maddo-review-pr13282.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr13282/environment) <!-- pix-api-maddo-review -->
- [ ] [Audit Logger](https://pix-audit-logger-review-pr13282.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr13282/environment) <!-- pix-audit-logger-review -->

> [!IMPORTANT]
> N'oubliez pas de déployer l'API pour pouvoir accéder aux fronts et/ou à l’API MaDDo.

JTEPIX committed:
Ajout tableau html (suppr tableau image)

JTEPIX committed:
Ajout retour à la ligne dans carousel

JTEPIX committed:
Update prompt-intermediaire.json

- supprimer les sauts de lignes entre paragraphes dans le carousel
- sortir le premier chatPix LLM du stepper
- retirer les doubles (tableau et carousel) suite à nouvelle version

JTEPIX committed:
doublons uuid

JTEPIX committed:
Correction input vide et instruction vide

pix-service commented:
:tada: This PR is included in version 5.191.0 :tada:

The release is available on [GitHub release](https://github.com/1024pix/pix/releases/tag/v5.191.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
